.github/resources/suppress_other_rmw.repos

```diff
@@ -0,0 +1,11 @@
+definitions:
+  - &empty_repo
+    type: zip
+    url: data:application/zip;base64,UEsFBgAAAAAAAAAAAAAAAAAAAAAAAA==
```

Comment:
To avoid the discussion if `COLCON_IGNORE` can be a directory or only a file, could this zip contain an empty file called `COLCON_IGNORE`?

Then `ros2/rosidl_typesupport_*/` should be used instead of `ros2/rosidl_typesupport_*/COLCON_IGNORE`.

Reply:
I tried that.

The problem with having a file is that I would need to overwrite the repo to put it in the right place. This breaks on Window: https://github.com/dirk-thomas/vcstool/issues/130